src/Symfony/Component/Cache/Traits/MemcachedTrait.php

```diff
@@ -84,7 +84,7 @@ private function init(\Memcached $client, string $namespace, int $defaultLifetim
      *
      * @throws \ErrorException When invalid options or servers are provided
      */
-    public static function createConnection($servers, array $options = [])
+    public static function createConnection(string | array $servers, array $options = [])
     {
         if (\is_string($servers)) {
             $servers = [$servers];
```

Comment:
couldn't comment on the correct line, but the `elseif` directly below this line can be removed.